### ISSUE TITLE
agogo theme doesn't display indented paragraphs correctly

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#356]
Because agogo.css_t does this at the very top:

```
  * {
     margin: 0px;
     padding: 0px;
  }
```

Indented paragraphs don't look indented.

To fix add the following (or something similar) somewhere **after** the above in agogo.css_t:

```
  blockquote {
     margin: 1em;
  }
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/356
- Originally reported by: [tpowers](https://bitbucket.org/tpowers)
- Originally created at: 2010-03-20T12:56:11.466

**Comment by shimizukawa** [comment]
_From [tpowers](https://bitbucket.org/tpowers) on 2010-03-21 14:25:57+00:00_

In addition add:
{{{
   ol {
      margin: 1.5em;
   }
}}}

Otherwise numbered lists stick out into left margin.

To be consistent you might also want to change:
{{{
   div.document ul {
     margin-left: 1.2em;
     list-style-type: square;
   }
}}}

to:
{{{
   div.document ul {
     margin: 1.5em;
     list-style-type: square;
   }
}}}

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-04-04 17:45:28+00:00_

Thanks, fixed in <<cset 15f3acc81ac419b127e0ca917402dcc049c8ed87>>.
